src/Encamina.Enmarcha.SemanticKernel.Connectors.Memory/Extensions/IServiceCollectionExtensions.cs

```diff
@@ -106,4 +106,71 @@ public static IServiceCollection AddSemanticTextMemory(this IServiceCollection s
                 .Build();
         });
     }
+
+    /// <summary>
+    /// Adds Azure AI Search as a named vector database for a memory store (<see cref="IMemoryStore"/>) to the <see cref="IServiceCollection"/> as a singleton service.
+    /// </summary>
+    /// <remarks>
+    /// This extension methods requires a <see cref="AzureAISearchOptions"/> to be already configured.
+    /// </remarks>
+    /// <param name="services"> The <see cref="IServiceCollection"/> to add services to.</param>
+    /// <param name="memoryProviderName">name of the memory provider.</param>
```

Comment:
Comentar es fastidioso, lo se, pero tratemos de tenerlo bien. Te faltó el artíclo. Es decir =>

`/// <param name="memoryProviderName">The name or key for the the memory provider.</param>`